src/test/ui/nll/ty-outlives/projection-where-clause-none.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(nll)]
+
+// Test that we are NOT able to establish that `<T as
```

Comment:
why is there no `projection-where-clause-none.stderr` in this PR?

Reply:
(it seems like a bunch of `.stderr` files have been left out of this PR?)

Reply:
hmm, curious, I have no such files locally...? I'll fix, anyway